### PR TITLE
remove execution_count on error messages

### PR DESCRIPTION
**Pull request by kojix2**
image

The `error` message type shouldn't have an `execution_count` field.
Thanks to IRKernel developers.
https://github.com/IRkernel/IRkernel/pull/564

### CODE REVIEW

File: lib/iruby/kernel.rb [hunk 1]
```diff
@@ -33,9 +33,7 @@ def create_backend
 
     def run
       send_status :starting
-      while @running
-        dispatch
-      end
+      dispatch while @running
```
Comment (mrkn) [inline lib/iruby/kernel.rb]: I don't want to include this change due to the following reasons.

1. I think the original form is more readable than the one-line form.
2. This change is not essential for the purpose of this pull-request.

Would you please revert this change?

Comment (kojix2) [reply lib/iruby/kernel.rb]: Ok. I will revert

Comment (kojix2) [reply lib/iruby/kernel.rb]: done. Thank you for your code revew.

Comment (mrkn) [reply lib/iruby/kernel.rb]: Please push your new commit.

Comment (kojix2) [reply lib/iruby/kernel.rb]: oh!

Comment (mrkn) [reply lib/iruby/kernel.rb]: Thanks!
